lib/resty/limit/req.lua

```diff
@@ -4,149 +4,145 @@
 local floor = math.floor
 local tonumber = tonumber
 
-
 local _M = { _VERSION = "0.01", OK = 1, BUSY = 2, FORBIDDEN = 3 }
 
+local function redis_connect(rds)
+  rds.host = rds.host or "127.0.0.1"
+  rds.port = rds.port or 6379
+  rds.timeout = rds.timeout or 1
+
+  local red = redis:new()
+
+  red:set_timeout(rds.timeout * 1000)
 
-local redis_limit_req_script_sha
-local redis_limit_req_script = [==[
-local key = KEYS[1]
-local rate = tonumber(KEYS[2])
-local now, interval = tonumber(KEYS[3]), tonumber(KEYS[4])
-local burst = tonumber(KEYS[5])
+  local ok, err
+  if rds.socket then
+      ok, err = red:connect(rds.socket)
```

Comment:
It is used here: https://github.com/hostinger/lua-resty-limit-req/blob/b7b48cd732a50f2bb7cc7827528711865e3973c0/lib/resty/limit/req.lua#L26